velox/docs/functions/spark/array.rst

```diff
@@ -196,7 +196,7 @@ Array Functions
 .. spark:function:: in(value, array(E)) -> boolean
 
     Returns true if value matches at least one of the elements of the array.
-    Supports BOOLEAN, REAL, DOUBLE, BIGINT, VARCHAR, TIMESTAMP, DATE input types.
+    Supports BOOLEAN, REAL, DOUBLE, BIGINT, VARCHAR, TIMESTAMP, DATE, DECIMAL input types.
```

Comment:
Perhaps document the decimal function in https://github.com/facebookincubator/velox/blob/9e418b172c7b316d351acd6c4b3428d5ca03eebd/velox/docs/functions/spark/decimal.rst. Thanks.